processing/src/test/java/org/apache/druid/segment/SchemalessIndexTest.java

```diff
@@ -64,9 +64,10 @@
 
 /**
  */
+@SuppressWarnings("SSBasedInspection") // static field(s) cannot be final because set in an static method
```

Comment:
`mergedIndex` doesn't need to be static. It seems to be that `index` doesn't need to be static, too. `getIncrementalIndex()` and `makeIncrementalIndex()` should become non-static, too.